tracer/build/_build/Build.Utilities.cs

```diff
@@ -452,6 +453,35 @@ private static MSBuildTargetPlatform GetDefaultTargetPlatform()
         return MSBuildTargetPlatform.x64;
     }
 
+    [CanBeNull]
+    private static string GetDefaultRuntimeIdentifier()
+    {
+        // https://learn.microsoft.com/en-us/dotnet/core/rid-catalog
+
+        var os = Platform switch
+                 {
+                     PlatformFamily.Windows => "win",
+                     PlatformFamily.Linux => "linux",
+                     PlatformFamily.OSX => "osx",
+                     _ => null
+                 };
+
+        var arch = (string)GetDefaultTargetPlatform() switch
+                   {
+                       "x86" => "x86",
+                       "x64" => "x64",
+                       "ARM64" or "ARM64EC" => "arm64",
+                       _ => null
+                   };
+
+        if (os is null || arch is null)
+        {
+            return null;
+        }
+
+        return $"{os}-{arch}";
```

Comment:
This gives a bunch of rids that we don't support, should we be more specific?
```suggestion
        var arch = (string)GetDefaultTargetPlatform()
        return Platform switch
               {
                   PlatformFamily.Windows => arch switch
                   {
                       "x86" => "win-x86",
                       "x64" => "win-x64",
                       _ => null,
                   },
                   PlatformFamily.Linux => arch switch
                   {
                       "x64" => "linux-x64",
                       "ARM64" or "ARM64EC" => "linux-arm64",
                       _ => null,
                   },
                   PlatformFamily.OSX => arch switch
                   {
                       "ARM64" or "ARM64EC" => "osx-arm64",
                       _ => null,
                   },
                   _ => null
               };
```

Reply:
Instead of nesting `switch` inside `switch`, I used:
```
(Platform , (string)GetDefaultTargetPlatform()) switch
{
   (Windows, "x86") => "win-x86",
   // ...
}
```

That ok?